tests/device_tests/test_msg_sd_protect.py

```diff
@@ -16,47 +16,36 @@
 
 import pytest
 
-from trezorlib import debuglink, device, messages as proto
+from trezorlib import device
 from trezorlib.exceptions import TrezorFailure
+from trezorlib.messages import SdProtectOperationType as Op
 
-from ..common import MNEMONIC12
+pytestmark = [pytest.mark.skip_t1, pytest.mark.sd_card]
 
 
-@pytest.mark.skip_t1
-class TestMsgSdProtect:
-    @pytest.mark.setup_client(mnemonic=MNEMONIC12)
-    def test_sd_protect(self, client):
+def test_sd_protect_enable(client):
+    # Disabling SD protection should fail
+    with pytest.raises(TrezorFailure):
+        device.sd_protect(client, Op.DISABLE)
 
-        # Disabling SD protection should fail
-        with pytest.raises(TrezorFailure):
-            device.sd_protect(client, proto.SdProtectOperationType.DISABLE)
+    # Enable SD protection
+    device.sd_protect(client, Op.ENABLE)
 
-        # Enable SD protection
-        device.sd_protect(client, proto.SdProtectOperationType.ENABLE)
+    # Enabling SD protection should fail
+    with pytest.raises(TrezorFailure):
+        device.sd_protect(client, Op.ENABLE)
 
-        # Enabling SD protection should fail
-        with pytest.raises(TrezorFailure):
-            device.sd_protect(client, proto.SdProtectOperationType.ENABLE)
 
-        # Wipe
-        device.wipe(client)
-        debuglink.load_device_by_mnemonic(
-            client,
-            mnemonic=MNEMONIC12,
-            pin="",
-            passphrase_protection=False,
-            label="test",
-        )
```

Comment:
Damn, I hadn't noticed this change previously. The purpose of the wipe is to make sure that wiping the storage while SD protection is enabled will not break the feature. The reason is that SD protection uses data stored on the SD card and data in the internal storage. Wiping the device removes the data in the internal storage but keeps the data on the SD card. Are you absolutely sure that this still get tested with this change?

Reply:
Well, let's make a separate test case for that.

Reply:
Why not just leave it?